registrations/tasks.py

```diff
@@ -31,6 +31,8 @@
 def send_registration_to_openhim(
     msisdn, referral_msisdn, channel, clinic_code, persal, sanc, timestamp
 ):
+    contact = get_rapidpro_contact(msisdn)  # Created in send_registration_to_rapidpro
+    uuid = contact.get("uuid")
```

Comment:
We can probably just return the UUID from the `send_registration_to_rapidpro` task to avoid the extra HTTP call, but not a big thing.